### PR TITLE
Revert "Merge pull request #306 from shareefalis/linphonecre"

### PR DESCRIPTION
**Pull request by ianblenke**
This reverts commit 4261b59ed7f9acabcdad8ad62912216df97b0009, reversing
changes made to bc1167a99123d159ceecde92b7ee2726fca05a03.

### CODE REVIEW

File: prepare.py [hunk 1]
```diff
@@ -517,7 +518,7 @@ def main(argv=None):
                 error("Could not clone tunnel. Please see http://www.belledonne-communications.com/voiptunnel.html")
                 return 1
         warning("Tunnel enabled, disabling GPL third parties.")
-        additional_args += ["-DENABLE_TUNNEL=ON", "-DENABLE_GPL_THIRD_PARTIES=OFF", "-DENABLE_FFMPEG=NO"]
+        additional_args += ["-DENABLE_TUNNEL=ON", "-DENABLE_GPL_THIRD_PARTIES=OFF"]
```
Comment (kennethdick) [inline prepare.py:521]: Keep track of this DENABLE_GPL_THIRD_PARTIES=OFF

File: prepare.py [hunk 2]
```diff
@@ -517,7 +518,7 @@ def main(argv=None):
                 error("Could not clone tunnel. Please see http://www.belledonne-communications.com/voiptunnel.html")
                 return 1
         warning("Tunnel enabled, disabling GPL third parties.")
-        additional_args += ["-DENABLE_TUNNEL=ON", "-DENABLE_GPL_THIRD_PARTIES=OFF", "-DENABLE_FFMPEG=NO"]
```
Comment (kennethdick) [inline prepare.py:520]: Why did we disable FFMPEG? -DENABLE_FFMPEG=NO

File: prepare.py (hunk 1 above)
Comment (ianblenke) [inline prepare.py:521]: This is for `--tunnel` builds, which we do not use.

File: prepare.py (hunk 2 above)
Comment (ianblenke) [inline prepare.py:520]: This is for `--tunnel` builds, which we do not use.

**Pull request merged**